tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -440,20 +445,45 @@ private void setTaskFamily(final List<String> listOfEcsTasksBeforeNewOneWasAdded
         }
     }
 
-    public void uploadRunInfo() {
-        runMetrics.setExecutionStatus(getExecutionStatus());
+    private void uploadRunInfo() {
+        extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().addRunExecutionsItem(runMetrics), Partner.AGC.name(), getEntryNameForApi(), version, "generated with tooltester ('run-workflows-through-wes' command)");
+    }
 
+    private void saveRunInfo() {
+        String fileName = Instant.now().toEpochMilli() + ".json";
+        final String filePath = resultDirectory + "/" + generateKey(getEntryNameForApi(), version, Partner.AGC.name(), fileName);
+        File runFile = new File(filePath);
+        Gson gson = new Gson();
+        runFile.getParentFile().mkdirs();
+        try {
+            runFile.createNewFile();
+        } catch (IOException e) {
+            exceptionMessage(e, "Error creating file", IO_ERROR);
+        }
+
+        try (
+                BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))
+        ) {
+            writer.write(gson.toJson(runMetrics));
+        } catch (IOException e) {
+            exceptionMessage(e, "There as an error writing to " + filePath, COMMAND_ERROR);
```

Comment:
Similar comment about exiting the program

Reply:
It now logs an error message if there is an error and continues on.